android/java/org/chromium/base/BravePreferenceKeys.java

```diff
@@ -1,9 +1,9 @@
 /* Copyright (c) 2020 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-package org.chromium.chrome.browser.preferences;
```

Comment:
What was the reason to change package ?

Reply:
We wants to access BravePreferenceKeys in `browser/download/internal/android/java/src/org/chromium/chrome/browser/download/BraveDownloadMessageUiControllerImpl.java` and from @samartnik's suggestion https://github.com/brave/brave-core/pull/16556#discussion_r1063447650